### ISSUE TITLE
Feature: XHTML output for EPUB

### ISSUE THREAD

**Issue by Artoria2e5** [KaTeX/KaTeX#2881]
**Is your feature request related to a problem? Please describe.**
I saw someone mentioning that [math support in epub is in a sad state](https://news.ycombinator.com/item?id=26694727). A quick search shows that MathML is the current only method of doing so, but we all know how painful it is to write, and on top of that not every reader is capable of rendering it.

Now the obvious solution is to use some front-end to convert something easier-to-write to MathML and some fallback -- precisely what KaTeX is designed to do. From some playing around on https://validator.nu, it looks like the only thing preventing validation with the `XHTML + SVG 1.1 + MathML 3.0` schema (where the unversioned XHTML refers to "HTML5") is a lack of `xmlns` on the `<svg>` element. Other asepcts of the generation are strict enough for XML as far as I can tell.

**Describe the solution you'd like:**
1. Add the attribute ` xmlns="http://www.w3.org/2000/svg"` to generated `<svg>` elements.
2. Preferably add a section in `node.md` talking about the XHTML usage.
3. (While we are at it, the demo and readme can use some mention on the MathML capability.)

**Link to or name of a (La)TeX package that provides the same feature:**
Not applicable.

**Describe alternatives you've considered:**
I originally considered requesting an HTML/XHTML output mode switch for (1), but the difference feels too small to justify it.

**Additional context:**
The EPUB people at https://github.com/w3c/epub-specs/issues/845 are talking about the mathml situation too. KaTeX's output fits into the fifth proposed method.

**Comment by edemaine** [comment]
This sounds like the motivation we needed for existing PR #2725.  I thought that XHTML was dead, but if it's still in use for the live format EPUB, and it's that easy to make the output compatible, it seems worth doing.

@Artoria2e5 Would you like to work on a documentation PR for your other points, assuming #2725 gets merged?

**Comment by Artoria2e5** [comment]
Heck yeah! I love writing documentation :)

**Comment by edemaine** [comment]
@Artoria2e5 I finally merged #2725 -- sorry for the delay. I checked that a simple document passed the XHTML + SVG 1.1 + MathML 3.0 schema. Would you like to write a PR to document this feature?

**Comment by chhantyal** [comment]
Looks like SVG output is already possible, just not documented? I would really appreciate if someone could post a short snippet.

**Comment by edemaine** [comment]
@chhantyal No, SVG output is not an option; see #375. What we're discussing here is the SVG that's a small part of the HTML output.
